snapcraft/parts/lifecycle.py

```diff
@@ -842,3 +840,38 @@ def _validate_and_get_partitions(yaml_data: Dict[str, Any]) -> Optional[List[str
         return project.get_partitions()
 
     return None
+
+
+def _is_manager(parsed_args: "argparse.Namespace") -> bool:
+    """Check if snapcraft is managing a build environment.
```

Comment:
```suggestion
    """Check if snapcraft is managing build environments.
```

Might make it a bit clearer, since the host instance can launch multiple managed runs